index.css

```diff
@@ -0,0 +1,30 @@
+body {
+    font-family: monospace;
+    background-color: lavender;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+header {
```

Comment:
To align the text in the middle via y-axis you could of used the `align-items:center` property